core/new-gui/src/app/app.module.ts

```diff
@@ -1,6 +1,7 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
 import { HttpClientModule } from '@angular/common/http';
+import { HttpModule } from '@angular/http';
```

Comment:
Why is `HttpModule` required in this case? 

According to the discussion here, https://stackoverflow.com/questions/45405593/whats-the-difference-between-httpmodule-and-httpclientmodule-in-angular2, `HttpModule` is deprecated and `HttpClientModule` is the new API that handles http requests. 

Is there any methods in the later PR that cannot use `HttpClientModule` but must use `HttpModule`?

Reply:
Thank you, Henry. I found that I was still using the outdated HttpModule and Http API for one of my service. I already updated my code and modified the test cases to remove all the HttpModule imports in my codes and remove the HttpModule from the `app.module.ts`